modules/social_features/social_event/modules/social_event_managers/src/Plugin/views/field/SocialEventManagersViewsBulkOperationsBulkForm.php

```diff
@@ -78,12 +88,14 @@ public function __construct(
     AccountInterface $currentUser,
     RequestStack $requestStack,
     EntityTypeManagerInterface $entity_type_manager,
-    ActionManager $pluginActionManager
+    ActionManager $pluginActionManager,
```

Comment:
Hi @nkoporec,
I have noticed one failing [check](https://github.com/goalgorilla/open_social/actions/runs/12157126315/job/33902986370?pr=4218) related to this change:
```Error: Method Drupal\social_event_managers\Plugin\views\field\SocialEventManagersViewsBulkOperationsBulkForm::__construct() invoked with 11 parameters, 12 required.```